spec/plaindate.html

```diff
@@ -943,20 +943,10 @@ <h1>
         1. If _operation_ is ~since~, let _sign_ be -1. Otherwise, let _sign_ be 1.
         1. Set _other_ to ? ToTemporalDate(_other_).
         1. If ? CalendarEquals(_temporalDate_.[[Calendar]], _other_.[[Calendar]]) is *false*, throw a *RangeError* exception.
-        1. Set _options_ to ? GetOptionsObject(_options_).
-        1. Let _disallowedUnits_ be « *"hour"*, *"minute"*, *"second"*, *"millisecond"*, *"microsecond"*, *"nanosecond"* ».
-        1. Let _smallestUnit_ be ? ToSmallestTemporalUnit(_options_, _disallowedUnits_, *"day"*).
-        1. Let _defaultLargestUnit_ be ! LargerOfTwoTemporalUnits(*"day"*, _smallestUnit_).
-        1. Let _largestUnit_ be ? ToLargestTemporalUnit(_options_, _disallowedUnits_, *"auto"*, _defaultLargestUnit_).
-        1. Perform ? ValidateTemporalUnitRange(_largestUnit_, _smallestUnit_).
-        1. Let _roundingMode_ be ? ToTemporalRoundingMode(_options_, *"trunc"*).
-        1. If _operation_ is ~since~, then
-          1. Set _roundingMode_ to ! NegateTemporalRoundingMode(_roundingMode_).
-        1. Let _roundingIncrement_ be ? ToTemporalRoundingIncrement(_options_, *undefined*, *false*).
```

Comment:
Is MaximumTemporalDurationRoundingIncrement missing here?

Reply:
In the status quo, DifferenceTemporalPlainDate does not call MaximumTemporalDurationRoundingIncrement but pass undefined to the second arg of ToTemporalRoundingIncrement directly. However, in this case, because disallowedUnits is « *"hour"*, *"minute"*, *"second"*, *"millisecond"*, *"microsecond"*, *"nanosecond"* » , therefore smallestUnit can only be one of "year", "month", "week" or "day" but no other unit. And we know MaximumTemporalDurationRoundingIncrement(smallestUnit) while smallestUnit is one of "year", "month", "week" , "day", always return undefined, therefore it is equivlent to pass in undefined directly.